modules/administration-guide/partials/con_supported-restic-compatible-backup-servers.adoc

```diff
@@ -0,0 +1,20 @@
+[id="supported-restic-compatible-backup-servers_{context}"]
+= Supported Restic-compatible backup servers
+
+{prod-short} uses the {prod-short} Operator and integrated Restic to back up and restore {prod-short} instances from backup snapshots on a configured backup server. The {prod-short} Operator automates the creation of a Restic backup repository on the backup server. To back up data, the {prod-short} Operator gathers the data required for a backup snapshot and uses Restic to create and manage the snapshot. To restore data, the {prod-short} Operator uses Restic to retrieve and decrypt the snapshot, and then the {prod-short} Operator applies the retrieved data to perform the recovery.
+
+{prod-short} can use the following backup servers that are compatible with the integrated Restic:
+
+SFTP:: See the documentation for the SFTP server solution you plan to use (link:https://www.openssh.com/[OpenSSH] or a derived commercial product) and the link:https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#sftp[Restic Docs on SFTP].
```

Comment:
I understand that I'm not a typical admin. That said, on another page, we recommend a specific REST server: https://github.com/restic/rest-server with which I ran into the following setting up issues:
1. Neither `restic` nor `rest server` show any error messages if something in the setup is missing or not as expected. So trying this combination to work turned into frustrating guesswork for me.
2. The README on https://github.com/restic/rest-server is brief and unclear, and I couldn't find online any third-party explanation about the details. I believe they need to improve their docs.
3. I was setting up the external server from scratch, so I ended up having to research and install some dependencies for the `rest server` that I didn't have to worry about for SFTP.
4. 
So I killed a lot of time trying to make the `rest server` on a remote server, and then I quickly and easily set up an SFTP server.
The page on REST servers is full-size in this PR. It's just that I moved it to the third place for users who have no preference and just pick the first option (like I did). If any users prefer to use `rest server`, they still have everything we expect to give them in the docs.

Reply:
@max-cx thank you for explaining the reasons. In my case, REST server was the easiest solution, it just worked, where as I had to deal with SSH keys and proper settings for SSH server in case of SFTP (not a problem, but took more time than for REST server).
@tolusha what do you think about this?